sdks/python/apache_beam/yaml/standard_io.yaml

```diff
@@ -112,3 +112,38 @@
     'WriteToJson': 'beam:schematransform:org.apache.beam:json_write:v1'
   config:
     gradle_target: 'sdks:java:extensions:schemaio-expansion-service:shadowJar'
+
+- type: renaming
+  transforms:
+    'ReadFromJdbc': 'ReadFromJdbc'
+    'WriteToJdbc': 'WriteToJdbc'
+  config:
+    mappings:
+      'ReadFromJdbc':
+        driver_class_name: 'driverClassName'
+        jdbc_url: 'jdbcUrl'
+        username: 'username'
+        password: 'password'
+        table_name: 'location'
+        read_query: 'readQuery'
+        num_rows: 'fetchSize'
```

Comment:
Is this something we want to exose?

Reply:
I think we need to expose the first 3 for sure. I thought about not including `num_rows`, but I thought it could be useful for some use-cases.

Upon thinking about it now though, that could be accomplished with a `LIMIT` query...

I'll go ahead and change it.